x/staking/keeper/validator.go

```diff
@@ -4,24 +4,32 @@ import (
 	"fmt"
 	"time"
 
+	"cosmossdk.io/math"
 	gogotypes "github.com/cosmos/gogoproto/types"
 
-	"cosmossdk.io/math"
+	"github.com/cosmos/cosmos-sdk/store"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/staking/types"
 )
 
+func (k Keeper) decodeValidator(bz []byte) (types.Validator, bool) {
+	var validator types.Validator
+	if bz == nil {
+		return validator, false
+	}
+	validator = types.MustUnmarshalValidator(k.cdc, bz)
+	return validator, true
+}
+
 // get a single validator
 func (k Keeper) GetValidator(ctx sdk.Context, addr sdk.ValAddress) (validator types.Validator, found bool) {
-	store := ctx.KVStore(k.storeKey)
+	st := ctx.KVStore(k.storeKey)
```

Comment:
ditto

Reply:
see (comment)[https://github.com/cosmos/cosmos-sdk/pull/13446#discussion_r997429703]
we're using `store` import here